feat(ui): ✨ Add profile section in explorer

marc-aurele-besner opened the pull request:
### User Profile

![iScreen Shoter - Google Chrome - 250402181852](https://github.com/user-attachments/assets/03078e93-3154-49fd-899e-31019e2ee781)
![iScreen Shoter - Google Chrome - 250402181917](https://github.com/user-attachments/assets/748a735e-42f5-45ae-a9e0-87994c06d505)

### Wallets

![iScreen Shoter - Google Chrome - 250402181950](https://github.com/user-attachments/assets/04b5e84c-5248-4b84-82e1-0b171093af02)
![iScreen Shoter - Google Chrome - 250402182017](https://github.com/user-attachments/assets/6df2a964-de79-4d9f-ba6b-5b426e277339)

### APIs

![iScreen Shoter - Google Chrome - 250402182039](https://github.com/user-attachments/assets/0736d1e6-8ea6-4df4-b102-a59fff5029ab)
![iScreen Shoter - Google Chrome - 250402182954](https://github.com/user-attachments/assets/fed7839a-0dfd-48ba-8ce3-8825b95c828e)


### Tags

![iScreen Shoter - Google Chrome - 250402182059](https://github.com/user-attachments/assets/84284259-c804-465e-b8f3-5180903cd5b0)
![iScreen Shoter - Google Chrome - 250402182156](https://github.com/user-attachments/assets/b75ed994-9521-46cb-be49-8c7de11be6e7)
![iScreen Shoter - Google Chrome - 250402183011](https://github.com/user-attachments/assets/c02a8b61-a38d-45c0-a325-434ba5417da4)



### User Banner and Profile pic

![iScreen Shoter - Google Chrome - 250402182224](https://github.com/user-attachments/assets/23512887-8fca-4c82-b390-510131c6e614)
![iScreen Shoter - Google Chrome - 250402183343](https://github.com/user-attachments/assets/7bfafea8-7152-4314-b855-edde70f7c887)


### Public Profile

![iScreen Shoter - Google Chrome - 250402182249](https://github.com/user-attachments/assets/ef1954e6-d4fc-4c8b-bf90-7f1d7d0a9f39)

### First time user

![iScreen Shoter - Google Chrome - 250402182347](https://github.com/user-attachments/assets/ad4c160f-fea1-411a-9efb-6f04b3a7be99)

marc-aurele-besner commented:
FYI I'm waiting to push to prod the current main branch before getting this PR review

iamnamananand996 commented:
Hi @marc-aurele-besner, adding a comment to notify you for PR review, as you were the owner of the PR so, can't able to add as reviewer.

marc-aurele-besner commented:
As I'm testing this branch, I'm realizing it's assuming the existence of some tables that don't actually exist in Hasura.
I'm going to add the necessary tables, relations, and metadata for these features and query to work.

iamnamananand996 commented:
> As I'm testing this branch, I'm realizing it's assuming the existence of some tables that don't actually exist in Hasura. I'm going to add the necessary tables, relations, and metadata for these features and query to work.

Yes, I think 🟢 does not have a `tags` table, need to create it. I was making use of 🟦 .

marc-aurele-besner commented:
I made some changes, noticeably,
- I removed the setting to set all wallets and tags public, and changed them to make the name and bio with public settings.
- I renamed profile `description` to `bio` in db and logic to fit the label
- I removed the concept of having multiple tags per wallet, the idea is to label wallets (that are not necessary link to your account) not to add multiple tags to them
- I added in the db a `is_public` for both wallets and tags, but these are not yet exposed by FE

clostao reviewed (approved):
LGTM
